gateways/stripe/espresso_stripe.js

```diff
@@ -0,0 +1,139 @@
+jQuery(document).ready(function($) {
+	
+	function ee3StripeElements() {
+		this.submitButtonId = '#ee-stripe-button-btn';
+		this.submitPaymentButton = {};
+		this.paymentIntentInput = {};
+		this.stripeErrorContainer = {};
+		this.stripeElements = {};
+		this.stripeCardElement = {};
+		this.cardHolderFirstName = {};
+		this.cardHolderLastName = {};
+		this.cardHolderAddress = {};
+		this.cardHolderAddress2 = {};
+		this.cardHolderCity = {};
+		this.cardHolderState = {};
+		this.cardHolderCountry = {};
+		this.cardHolderZip = {};
+		this.initialized = false;
+
+		/**
+		 * @function initialize
+		 */
+		this.initialize = function() {
+			this.initialize_objects();
+		
+			// ensure that the Stripe js class is loaded
+			if ( typeof Stripe === 'undefined' ) {
+				return;
+			}
+
+			this.stripe = Stripe( ee_stripe_args.stripe_pk_key );
+			const args = {};
+			if ( ee_stripe_args.data_locale ) {
+				args.locale = ee_stripe_args.data_locale;
+			}
+			this.stripeElements = this.stripe.elements( args );
+			this.stripeCardElement = this.stripeElements.create( 'card' );
+			this.stripeCardElement.mount( '#ee-stripe-card-element' );
+			
+			// Set up listener for payment button
+			this.set_listener_for_submit_payment_button();
+
+			//alert('this.initialized');
+			this.initialized = true;
+		};
+
+		/**
+		 * @function initialize_objects
+		 */
+		this.initialize_objects = function() {
+			//console.log( JSON.stringify( '**this.initialize_objects**', null, 4 ) );
+			this.submitPaymentButton = $( this.submitButtonId );
+			this.paymentIntentInput = $( '#espresso_stripe_payment_intent_id' );
+			this.stripeErrorContainer = $( '#espresso_stripe_errors' );
+			this.cardHolderFirstName = $( '#espresso_stripe_first_name' );
+			this.cardHolderLastName = $( '#espresso_stripe_last_name' );
+			this.cardHolderEmail = $( '#espresso_stripe_email' );
+			this.cardHolderAddress = $( '#espresso_stripe_address' );
+			this.cardHolderCity = $( '#espresso_stripe_city' );
+			this.cardHolderState = $( '#espresso_stripe_state' );
+			this.cardHolderCountry = $( '#espresso_stripe_country' );
+		};
+
+		/**
+		 * @function set_listener_for_submit_payment_button
+		 */
+		this.set_listener_for_submit_payment_button = function() {
+			const stipe_instance = this;
+			//console.log( 'initialize', 'set_listener_for_submit_payment_button', true );
+			this.submitPaymentButton.on( 'click', function(e) {
+				e.preventDefault();
+				const billingDetails = {};
```

Comment:
:police_car:  "I'm sorry sir, are you allowed to use that `const`? Let me see your build process that uses Babel.js to make it compatible with older browsers. What's that? You aren't using a build process here? Tsk tsk..."

Should probably just use `var` if we want this to be compatible with older browsers. Sorry. (FYI I'm speaking from having made this sort of mistake myself recently.)